### PR TITLE
Add nokia3310

### PR DESCRIPTION
**Pull request by pkong-ds**
<details>

<summary> Outdated </summary>

## TODO
- Fix `upload image here` cropped
- make mask image look better (see below, have some shiny edges currently)
image

## Preview 

https://github.com/user-attachments/assets/e7b8caa0-08b7-4cb5-8265-835f3a6bbd61

</details>

## Preview

https://github.com/user-attachments/assets/246dd4dc-abc2-44eb-bc6c-4d2f689c6177

### CODE REVIEW

File: src/scripts/device_info.json [hunk 1]
```diff
@@ -6870,6 +6870,31 @@
...
       "available_perspectives": ["Portrait", "Landscape"]
+    },
+    {
+      "credits": "<p><a href=\"https://www.skymakers.digital/\" target=\"blank\">Skymaker's Devleoper</a></p>",
+      "color_str": "Default",
+      "color": null,
+      "meta_title": "Nokia 3310 Mock Up",
+      "meta_description": "1 click to generate your Nokia 3310 mockup! You can wrap screenshots in Nokia 3310 for prototypes.",
+      "display_resolution": [610, 433],
+      "device_type": "TV",
```
Comment (chankiyu22) [inline src/scripts/device_info.json]: Unbreakable? :rabbitsmile:

**Comment by pkong-ds** [comment]
@chankiyu22 can help merge this? thanks!

**Comment by pkong-ds** [comment]
@chankiyu22 updated, thanks!

File: src/scripts/device_info.json (hunk 1 above)
Comment (pkong-ds) [reply src/scripts/device_info.json]: maybe too much work if need to change design 🐶 let's do it when we free-er~

**Review by chankiyu22 — APPROVED** [review]

**Pull request merged**